build.gradle.kts

```diff
@@ -29,3 +33,9 @@ tasks.register<Exec>("installGitHooks") {
 
 val initialTaskNames: List<String> = project.gradle.startParameter.taskNames
 project.gradle.startParameter.setTaskNames(initialTaskNames + listOf("installGitHooks"))
+
+tasks.register("detektAll").configure {
```

Comment:
Why do we need register the detektAll if we register it in the convention plugin

Reply:
it's a chiken and egg problem. detektAll registered inside convention plugin of this project, but it needs detekt task itself